src/libsync/vfs/cfapi/cfapiwrapper.cpp

```diff
@@ -321,8 +440,10 @@ OCC::Result<void, QString> OCC::CfApiWrapper::registerSyncRoot(const QString &pa
     }
 }
 
-OCC::Result<void, QString> OCC::CfApiWrapper::unegisterSyncRoot(const QString &path)
+OCC::Result<void, QString> OCC::CfApiWrapper::unregisterSyncRoot(const QString &path)
 {
+    Q_ASSERT(deleteSyncRootRegistryKey(path));
```

Comment:
Ditto, won't be executed in release builds

Reply:
> Ditto, won't be executed in release builds

@er-vin Fixed.